src/jit/CMakeLists.txt

```diff
@@ -191,6 +196,10 @@ if (CLR_BUILD_JIT32)
   set(JIT_BASE_NAME ryujit)
 endif()
 
+if(WIN32)
+  add_definitions(-DFX_VER_INTERNALNAME_STR=${JIT_BASE_NAME}.dll)
```

Comment:
Here is what I see.
![image](https://cloud.githubusercontent.com/assets/10963908/15758456/8901d5e8-28bf-11e6-8e88-4d86baee3740.png)